src/patternfly/components/Toolbar/examples/Toolbar.md

```
@@ -557,6 +557,246 @@ Several components in the following examples do not include functional and/or ac
 {{/toolbar}}
 ```
 
+## Toolbars responsive to their container
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)